Dockerfile

```
@@ -28,9 +28,9 @@ RUN yum update --assumeyes \
 
 # Install the Python packages necessary to build a deployment package.
 RUN python3 -m pip install --no-cache-dir \
-    pip \
-    setuptools \
-    wheel \
+  pip \
+  setuptools \
+  wheel \
```

Did the spacing on these need to be changed?

```suggestion
    pip \
    setuptools \
    wheel \
```

---

That is how VS Code autoformatted it and I actually agree here.  I think `skeleton-aws-lambda-python` should be modified to use this spacing, which aligns with the rest of the Dockerfile (i.e. continued lines are indented two spaces).  I don't see why these particular lines (and only these) should be indented four spaces.

---

They are indented four spaces because to separate that they are arguments to a command since this Docker instruction chains commands:
https://github.com/cisagov/publish-egress-ip-lambda/blob/ac3ddf120769e0932628fceacf7b172329599c4c/Dockerfile#L30-L34
Just as a refresher it chains commands to ensure that `pipenv` (and its dependencies) have the latest versions of `pip`, `setuptools`, and `wheel` available when installing.

---

As a thought the syntax is borrowed from more traditional Docker projects that pin these so we could just do
```dockerfile
RUN python3 -m pip install --no-cache-dir pip setuptools wheel
  && python3 -m pip install --no-cache-dir pipenv
```
here and in the skeleton to negate the issue entirely.

---

> As a thought the syntax is borrowed from more traditional Docker projects that pin these so we could just do
> 
> ```dockerfile
> RUN python3 -m pip install --no-cache-dir pip setuptools wheel
>   && python3 -m pip install --no-cache-dir pipenv
> ```
> 
> here and in the skeleton to negate the issue entirely.

I like this solution the best- see https://github.com/cisagov/publish-egress-ip-lambda/pull/4/commits/29fbe09a56affdeac2aca9eeb88f3b01e5f43152.